src/main/index.ts

```diff
@@ -106,11 +106,13 @@ app.on("ready", async () => {
   await Promise.all([
     userStore.load(),
     clusterStore.load(),
-    workspaceStore.load(),
     extensionsStore.load(),
     filesystemProvisionerStore.load(),
   ]);
 
+  // load this after clusterStore, because it does validation on its entries
+  await workspaceStore.load();
```

Comment:
Can you keep it in the `promise.all()` as `clusterStore.load().then(workspaceStore.load)`?

Reply:
Yes I think so